examples/todoapp/repositories/ItemRepository.js

```diff
@@ -3,17 +3,26 @@ const { Ok, Err } = require('../../../src/buchu')
 class ItemRepository {
 
 	constructor() {
+		this._list = [
```

Comment:
Is this file part of the solution?

Reply:
not really, I just thought about implementing a test Use Case to give an example using the new feature, was too much?